Library/Homebrew/dev-cmd/install-bundler-gems.rb

```diff
@@ -15,14 +15,19 @@ def install_bundler_gems_args
       description <<~EOS
         Install Homebrew's Bundler gems.
       EOS
+      comma_array "--groups=",
+                  description: "Installs the specified comma-separated list of gem groups (default: last remembered)."
```

Comment:
```suggestion
                  description: "Installs the specified comma-separated list of gem groups (default: last used)."
```

(assuming that's correct?)